### DISCUSSION TITLE
Technical Comparison of GrapeJS, Mosaico, MailChimp, etc?

### DISCUSSION THREAD

**Discussion by dm17** [GrapesJS/grapesjs#4146] [Q&A]
Maybe this sounds basic, but it is quite opaque to me... For newsletters and the like, these editors like grapesjs and services like MailChimp all claim to add special HTML that allows emails to be more likely compatible with the various email clients people might be using.
I have two issues:
-- It isn't obvious how various WYSIWYG editors compare in terms of compatibility with email clients
-- It isn't obvious how the various editors compare in terms of compatibility with SMTP relay / senders (Sparkpost, SendGrid, SES, PostMark, etc)

Can anyone help me clarify this, or point me to some resources that will shed light on these issues?

**Comment by artf** [comment]
Hi @dm17 GrapesJS has nothing to do with email/newsletter itself and it doesn't make sense to compare it with other applications/platforms you're mentioning. GrapesJS is a simple client JS framework for building web-based builders. So you can build the website/newsletter/document/etc. builder but it all depends on the developer who implements the framework and its knowledge on creating proper components for the specific target environment (browsers, email clients, etc.)
